### DISCUSSION TITLE
RouteYou

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#586] [Q&A] [answered]
RouteYou (premium)web exports compatible gpx navigation tracks for Cruiser and Locus. 
[RouteYou](https://www.routeyou.com/en-gb)
Save your design in RouteYou and select download mode gpx Locus map
The following RouteYou instructional videos show how to avoid or place u-turns.

Nl: https://youtu.be/LNw4NVgN1qY?si=Qurj5hNwgq0o8cKN
EN: https://youtu.be/gvING6mWeoU?si=odhOsSAWSsqZc5GQ
FR: https://youtu.be/2QPrFhLLeD0?si=GSjI_ZcY_nudJpQh
DE: https://youtu.be/ujR5tM0Ppow?si=0H-LwKM5K6RkpLSX

**Comment by devemux86** [comment]
@0709wiwiwi Please do not close the discussions because then they will be hidden.

**Reply by 0709wiwiwi** [reply, marked as answer]
RouteYou is not relevant anymore for Cruiser usage. 
Future gpx export will only be offered in the subscription version.
Even better would be to completerly remove the Routeyou links.

**Reply by devemux86** [reply]
@0709wiwiwi Then community should be informed about the RouteYou change and leave the discussion for future reading.
